src/subscription_manager/managercli.py

```diff
@@ -743,7 +743,10 @@ def _do_command(self):
                 return
 
         try:
-            self._validate_options()
```

Comment:
I think it is not needed to be called, because `_validate_options()` uses `identity.is_valid()` for checking validity of identity and it is called at line 747.